org.eclipse.buildship.core.test/src/main/groovy/org/eclipse/buildship/core/util/binding/PropertyTest.groovy

```diff
@@ -0,0 +1,138 @@
+package org.eclipse.buildship.core.util.binding
+
+import spock.lang.Specification
+
+import com.google.common.base.Optional
+
+class PropertyTest extends Specification {
```

Comment:
⭕️ Where do we still use Property?

Reply:
In [`ProjectImportConfiguration`](https://github.com/eclipse/buildship/blob/1ffb28156481e2e83cf9be768ad5e008c76fe81c/org.eclipse.buildship.core/src/main/java/org/eclipse/buildship/core/projectimport/ProjectImportConfiguration.java).